Sources/GDSCommon/UI/UIKit/SharedViews/Buttons/ButtonViewModel.swift

```diff
@@ -2,11 +2,16 @@ import UIKit
 
 public protocol ButtonViewModel {
     var title: GDSLocalisedString { get }
-    var icon: String? { get }
+    var icon: ButtonIconViewModel? { get }
     var shouldLoadOnTap: Bool { get }
     var action: () -> Void { get }
 }
 
+public protocol ButtonIconViewModel {
```

Comment:
please can this go in a file of its own?

Reply:
Sure :) will change that now